tmdb/parsers/seasons.py

```diff
@@ -10,23 +11,19 @@ class SeasonParser(Parser[Season]):
 
     object_class = Season
 
-    @staticmethod
-    def _parse_episode(episode: dict) -> Episode:
-        number: int = episode['episode_number']
-        synopsis: str = episode['overview']
-        return Episode(number=number, synopsis=synopsis)
-
     def _get_list_episodes(self, data: dict) -> List[Episode]:
         episodes: List[dict] = data['episodes']
         list_episodes = []
         for episode in episodes:
-            list_episodes.append(self._parse_episode(episode))
+            episode_parser = EpisodeParser()
```

Comment:
You can also create a single parser a reuse it in the `for` loop? Then it can even be condensed to a one-liner:

```python
list_episodes = [parser.parse(episode) for episode in episodes]
```